documentation/puppetdb-faq.markdown

```diff
@@ -6,7 +6,7 @@ canonical: "/puppetdb/latest/puppetdb-faq.html"
 ---
 
 [trouble_kahadb]: ./trouble_kahadb_corruption.html
-[migrating]: ./migrate.html
+[migrating]: ./anonymization.html
```

Comment:
So anonymization.html, when you open it describes itself as instructions for anonymizing data, yes its also got some import/export commands, but doesn't break down a migrate like the old migrate.html did ... its examples are even inclusive of anonymization modes which isn't what a user would expect I guess. I think, either the anonymization doc needs a better migration (with no anon) change, or we should resurrect migrate.html but leave out the storeconfig stuff.